code/modules/clothing/glasses/glasses.dm

```diff
@@ -502,6 +502,7 @@
 	desc = "An Ahdominian made veil that allows the user to see while obscuring their eyes."
 	icon_state = "tajblind"
 	item_state = "tajblind"
+	flags_cover = GLASSESCOVERSEYES
 	flags_inv = HIDEEYES

```

Comment:
Ok, so this was my bad, apparently HIDEEYES is not the flag we wanted here. HIDEEYES is meant for masks and helmets and is used to hide the glasses slot from examines... the GLASSESCOVEREYES flag is probably what you wanted the whole time.